### ISSUE TITLE
Tree view no longer scrollable.

### ISSUE THREAD

**Issue by thirashima** [olmokramer/atom-autohide-tree-view#68]
Seems like scrollability of the tree view got broken somehow with the release of Atom 1.6. Switching themes and reloading editor does not seem to fix it.

**Comment by spencerthayer** [comment]
I can confirm I've experianced this bug for a while now. In fact the whole plugin is so buggy (sometimes doesn't show the tree at all, won't scroll, will not even update tree element names when changes happen) that I've uninstalled it and likely will never use it again.

**Comment by davidhantson** [comment]
Same as @spencerthayer , it's so buggy that I often have to restart atom or re-install the plugin. The non-scrollable area makes it really unusable. Any ideas on how to quick fix this?

**Comment by cwg999** [comment]
@davidhantson Using the "Push Editor" option scrolling works fine for me.

**Comment by davidhantson** [comment]
@cwg999 that is a nice workaround thank you.

**Comment by AustP** [comment]
@davidhantson 
Here's a workaround without using the "Push Editor" option:

Add the following to your custom stylesheet (Edit > Stylesheet...)

```
.tree-view-resizer {
    border-right: 1px solid @overlay-border-color;
    bottom: 0px;
    top: 0;
}
```

**Comment by rm-code** [comment]
This seems to be fixed for me in 1.8.0-beta3. Can anybody confirm?

**Comment by rainbow-pooping-unicorn** [comment]
same issue on my site... (1.8.0)

**Comment by calebswank11** [comment]
Realize this is an old post, but just ran into this today and gave the parent container an overflow of scroll. this way you can scroll at any time without interference.

```
.tool-panel {
  overflow: scroll;
}
```

**Comment by spencerthayer** [comment]
Please fix this in the main.

**Comment by thillmfg** [comment]
@calebswank11 This worked for me in 1.17.0-beta5
Thank you

**Comment by HauntedSmores** [comment]
Randomly had this issue on Win10.
```
.tool-panel {
  overflow-y: auto;
}
```
This worked well for me.

Slight modification of @calebswank11 's answer

**Comment by christopherdiehl** [comment]
Issue on Atom v1.7 on Mac OsX Sierra

**Comment by calebswank11** [comment]
@HauntedSmores might be best to leave as straight `overflow: scroll` in the event you get a large tree, with a framework like angular or similar, you will need/desire the capability to scroll-x as well.

**Comment by HauntedSmores** [comment]
@calebswank11 Good point. I was just thinking that the draggable panel would be used for that instead (I use a mouse with 1 directional scroll (y-axis, obviously) and I would never click-drag the x-axis scroll bar). 

The more significant change though, for me, was really just `overflow: auto` rather than `scroll` so it gets rid of the scroll bars when its unnecessary. Good looks on finding the right selector though. This should def be fixed asap

**Comment by Glavin001** [comment]
I have **not installed atom-autohide-tree-view**, however I am experiencing this issue. Does anyone know how this could be?

![image](https://cloud.githubusercontent.com/assets/1885333/26534671/eff6bb02-43fc-11e7-85ef-46ab57744209.png)


---

The suggested workarounds above work! Thank you!

# Steps
To solidify the steps I took:

1. Go to menu `Atom -> Stylesheets...`. It should open `styles.less`.
OR
You can use the Command Palette:

![image](https://cloud.githubusercontent.com/assets/1885333/26534617/83951b2a-43fc-11e7-9052-11e8f25d03e8.png)

2. Add the following styles:

```less
.tree-view {
  &.tool-panel {
    overflow: auto;
  }
}
```

![image](https://cloud.githubusercontent.com/assets/1885333/26534623/8fc9c580-43fc-11e7-9e4d-1bd11212a330.png)


3. Reload/Restart Atom

![image](https://cloud.githubusercontent.com/assets/1885333/26534627/9a334f28-43fc-11e7-8abb-1d8814a16e18.png)

## Before

Notice the missing scroll bar.

![image](https://cloud.githubusercontent.com/assets/1885333/26534654/d0a2d8f8-43fc-11e7-93cd-ad6991e78218.png)


## After

Notice the new scrollbar! 😃 

![image](https://cloud.githubusercontent.com/assets/1885333/26534661/d8b03612-43fc-11e7-9bea-d690c470dbc2.png)

**Comment by esappear** [comment]
【command + option + I 】open the delevop tool and change the style of the panel to 【overflow: scroll】 manually.

**Comment by heyakyra** [comment]
@cwg999 what's the "push editor" option? 

@Glavin001 I have this problem with nuclide and no other packages so I'm going to link there in case anyone from google ends up here:  https://github.com/facebook/nuclide/issues/1532

**Comment by olmokramer** [comment]
Hi everyone, I'm sorry for the extremely late reply. As you have probably noticed I've stopped maintaining my Atom plugins a few years ago. I got very frustrated with how Atom was maintained and started ignoring all of my plugins. I'm sorry for that. I won't install Atom to test and mergeer PRs, but I'd be happy to transfer my repositories to anyone that wants to maintain any of my plugins. Again, I'm sorry.

**Comment by wonko-wedler** [comment]
It was promising for a while, but meanwhile to me it's time to move on.
Navigation in files is a pain in the ass - seems the only way is the middle button which reactivates scrolling ... for a while.
Foreground color for selections is still not working, which reduces the choice of selection solor dramatically and auto-completion is totally blind when rearranging parts or working in regex.
The effort of the plugin authors is really appreciated, and most plugins do a good job for certain situations, but without any core that ensures a save level of usability I prefer a stable editor and spending effort for some tools.

**Comment by heyakyra** [comment]
My next editor: http://kakoune.org/why-kakoune/why-kakoune.html
